iidm/iidm-mergingview/src/main/java/com/powsybl/iidm/mergingview/MergedLine.java

```diff
@@ -0,0 +1,401 @@
+/**
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.iidm.mergingview;
+
+import com.powsybl.commons.PowsyblException;
+import com.powsybl.commons.extensions.Extension;
+import com.powsybl.iidm.network.*;
+import com.powsybl.iidm.network.util.LimitViolationUtils;
+
+import java.util.Collection;
+import java.util.List;
+import java.util.Objects;
+import java.util.Set;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+/**
+ * @author Thomas Adam <tadam at silicom.fr>
+ */
+class MergedLine implements Line {
+
+    private final MergingViewIndex index;
+
+    private final DanglingLine dl1;
+
+    private final DanglingLine dl2;
+
+    private String id;
+
+    MergedLine(final MergingViewIndex index, final DanglingLine dl1, final DanglingLine dl2) {
+        this.index = Objects.requireNonNull(index, "merging view index is null");
+        this.dl1 = Objects.requireNonNull(dl1, "dangling line 1 is null");
+        this.dl2 = Objects.requireNonNull(dl2, "dangling line 2 is null");
+        this.id = buildId(dl1, dl2);
```

Comment:
You have to ensure the unicity of this ID (or throw depending on the policy)

Reply:
How can i get the current policy here ? 
No ensureIdUnicity field is given here ...